api/api.go

```diff
@@ -77,6 +78,31 @@ func (response GQLResponseErrors) ToError() error {
 	return errors.New(strings.Join(messages, ": "))
 }
 
+// GraphQLServerAddress returns the full address to CircleCI GraphQL API server
+func GraphQLServerAddress() (string, error) {
+	// 1. Parse the endpoint
+	endpoint, err := url.Parse(viper.GetString("endpoint"))
```

Comment:
Pull `"endpoint"`/`"host"` into a const?

Reply:
Since this is the only place which types them out, I'm deciding against this refactor for now.

If it turns out we end typing these in more places then we can make that call.